src/plugins/intel_cpu/src/utils/debug_capabilities.cpp

```diff
@@ -198,8 +198,7 @@ std::ostream & operator<<(std::ostream & os, const Node &c_node) {
                     replace_all(shape_str, " ", "");
                     leftside << comma << desc->getPrecision().get_type_name()
                                 << "_" << desc->serializeFormat()
-                                << "_" << shape_str
-                                << "_" << ptr->getData();
```

Comment:
suggest keep it and work around this way like -
<< "_" << ptr->getSize() > 0 ? ptr->getData() : "invalid"

Reply:
I will try.

Reply:
Updated.
1: When we call `getSize()` here, sometimes trigger new issue:
`Can't get memory size for undefined shape`
2: `typeid().name` only can current variable type, for example: we only get based class type here.
3: I add `hasActualMemory()` to judge if we can call `getData()`;

Reply:
I prefer #1.  We can check if the shape is static first.

Reply:
As you said, `getData` is essential, if the shape is static, don't we print `getData`? @ceciliapeng2011

Reply:
Catch this exception, and then print "?", it seems reasonable and it has smaller changes. @ceciliapeng2011 is it OK?